README.md

```diff
@@ -1 +1,5 @@
 # 🚁 < 🛸
+
+cd post-conveter-script
```

Comment:
typo: post-conveter-script -> post-converter-script

Reply:
Good catch, it's fixed.